blocks/mosaic-reveal/mosaic-reveal.css

```diff
@@ -0,0 +1,215 @@
+.mosaic-reveal-container {
+  display: flex;
```

Comment:
We don't want to couple the mosaic-reveal to this behaviour, can you implement it with the selector"column-section" (https://github.com/aemsites/esri/issues/73) instead? (it's added via column metadata, you can put the code in styles.css)

Reply:
So I should add
```
  display: flex;
  justify-content: center;
  align-items: center;
```
to `.column-section`?